File: data/python/MAXIMIZE_VOLUME_CUBOID_GIVEN_SUM_SIDES_1_MAIN.py
if __name__ == '__main__':
    param = [
    (8,),
    (96,),
    (96,),
    (96,),
    (12,),
    (95,),
    (72,),
    (81,),
    (42,),
    (13,)
        ]
    n_success = 0
    print("class,mutant,test_index,result\n")
    for i, parameters_set in enumerate(param):
        try:
            print("MAXIMIZE_VOLUME_CUBOID_GIVEN_SUM_SIDES_1," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("MAXIMIZE_VOLUME_CUBOID_GIVEN_SUM_SIDES_1," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/MAXIMIZE_VOLUME_CUBOID_GIVEN_SUM_SIDES_1_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(8);
    param0.add(96);
    param0.add(96);
    param0.add(96);
    param0.add(12);
    param0.add(95);
    param0.add(72);
    param0.add(81);
    param0.add(42);
    param0.add(13);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("MAXIMIZE_VOLUME_CUBOID_GIVEN_SUM_SIDES_1," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("MAXIMIZE_VOLUME_CUBOID_GIVEN_SUM_SIDES_1," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
